ci/docker_ci.sh

```diff
@@ -8,7 +8,14 @@ set -e
 DOCKER_IMAGE_PREFIX="${DOCKER_IMAGE_PREFIX:-envoyproxy/envoy}"
 
 # "-google-vrp" must come afer "" to ensure we rebuild the local base image dependency.
-BUILD_TYPES=("" "-alpine" "-alpine-debug" "-google-vrp")
+if [ $(uname -m) == "aarch64" ]; then
+    # There isn't alpine-glibc image for arm platform, so do not build alpine based image.
+    BUILD_TYPES=("" "-google-vrp")
```

Comment:
I don't think we need `google-vrp` variant either, @htuch?

Reply:
Yeah, no VRP for ARM.